sofria_mediaId.js

```diff
@@ -78,7 +78,7 @@ const generateJsonContentByUSXFile = function (usxFile, jsonPathOutput) {
         }
       };
 
-      let poolChapter = new PromisePool(promiseChapterProducer, 4);
+      let poolChapter = new PromisePool(promiseChapterProducer, 6);
```

Comment:
MAGIC NUMBER -- please create constants for the PromisePool parameters so it is easier to understand what it represents

Reply:
@bradflood , I have created two new constants related to PromisePool concurrency:

```javascript
const NUM_USX_FILES_TO_PROCESS_ASYNC = 2;
const NUM_CHAPTERS_TO_PROCESS_ASYNC = 6;